{"name": "test_try_signup_with_invalid_username", "status": "broken", "statusDetails": {"message": "AttributeError: 'str' object has no attribute 'spllit'. Did you mean: 'split'?", "trace": "item = <Function test_try_signup_with_invalid_username>\n\n    @pytest.hookimpl(hookwrapper=True, trylast=True)\n    def pytest_runtest_call(item: Item):\n        yield\n>       allure.dynamic.title(\" \".join(item.name.spllit(\"_\")[1:]).title())\n                                      ^^^^^^^^^^^^^^^^\nE       AttributeError: 'str' object has no attribute 'spllit'. Did you mean: 'split'?\n\ntests\\conftest.py:114: AttributeError"}, "start": 1751799952520, "stop": 1751799952521, "uuid": "ebed630e-6ec8-4d03-8e4b-32805a3f0d43", "historyId": "5bf71bf0cb3efa99e9ce6fbe2d86867e", "testCaseId": "5bf71bf0cb3efa99e9ce6fbe2d86867e", "fullName": "tests.test_auth.TestsSignup#test_try_signup_with_invalid_username", "labels": [{"name": "epic", "value": "Форма регистрации"}, {"name": "story", "value": "Попытка регистрации под невалидным именем пользователя"}, {"name": "feature", "value": "Регистрация"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_auth"}, {"name": "subSuite", "value": "TestsSignup"}, {"name": "host", "value": "WIN-V8TQK9SAURU"}, {"name": "thread", "value": "11924-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_auth"}]}